packages/shared/routes/onboarding/views/ledger-setup/views/ConnectView.svelte

```diff
@@ -47,28 +42,22 @@
     }
 
     function handleContinueClick(): void {
```

Comment:
Small detail, but I've been renaming all of these click handler functions to `onContinueClick` or `onBackClick` to help readability with the `OnboardingLayout` component props.